### ISSUE TITLE
Fix mobile layout issues

### ISSUE THREAD

**Issue by jlegrone** [L4GG/timeline#22]
Media captions in particular are an issue, this may be fixed by upgrading our TimelineJS3 dependency.

**Comment by thekevinscott** [comment]
@jlegrone Just to clarify (from your message on Slack) - should I hold off on working on this until you upgrade TimelineJS3? Or do you want me to take a stab at upgrading TimelineJS3?

**Comment by thekevinscott** [comment]
I see we are referencing a forked version of the original software - implying there are probably some custom changes I'm not privy to, so I'm not going to try upgrading just now.

**Comment by jlegrone** [comment]
@thekevinscott I need to work on how I receive my github notifications, sorry!  I'm closing this as I believe the issues I noticed were caused by opening the timeline in a desktop browser and then switching to an emulated mobile view.  Can't replicate on my device.

For the record, as of now we haven't made any modifications to the L4GG fork of TimelineJS3 -- I just added that dependency in anticipation of such modifications.
